bugbot/rules/lot_of_see_also.py

```diff
@@ -0,0 +1,55 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from bugbot import utils
+from bugbot.bzcleaner import BzCleaner
+
+
+class SeveralSeeAlso(BzCleaner):
+    def __init__(self, nweeks: int = 52, see_also: int = 5):
+        """Constructor
+
+        Args:
+            nweeks: the maximum number of weeks from the submission date
+            see_also: the number of 'see also' in the bug
+        """
+        super().__init__()
+        self.nweeks = nweeks
+        self.see_also = see_also
+
+    def description(self):
+        return "Bugs with several see_also for the last {} weeks".format(self.nweeks)
+
+    def columns(self):
+        return ["id", "summary", "creation", "last_change", "see_also_count"]
+
+    def handle_bug(self, bug, data):
+        bugid = str(bug["id"])
+        see_also_count = bug["see_also_count"]
```

Comment:
It seems that `see_also_count` can be used as a search field only.
```suggestion
        see_also_count = len(bug["see_also"])
```

nit: it could be assigned to the bug dict directly since we are not using the variable.